ark/utils/data_utils_test.py

```diff
@@ -456,3 +457,74 @@ def test_crop_image_stack():
         crop_input.shape[2] / crop_size) * (1 / stride_fraction) * (1 / stride_fraction)
 
     assert np.array_equal(cropped.shape, (num_crops, crop_size, crop_size, crop_input.shape[3]))
+
+
+def test_combine_point_directories():
+    # first test the case where the directory specified doesn't exist
+    with pytest.raises(ValueError):
+        data_utils.combine_point_directories(os.path.join("path", "to", "undefined", "folder"))
+
+    # now we do the "real" testing...
+    with tempfile.TemporaryDirectory() as temp_dir:
+        os.mkdir(os.path.join(temp_dir, "test"))
+
+        os.mkdir(os.path.join(temp_dir, "test", "subdir1"))
+        os.mkdir(os.path.join(temp_dir, "test", "subdir2"))
+
+        os.mkdir(os.path.join(temp_dir, "test", "subdir1", "point1"))
+        os.mkdir(os.path.join(temp_dir, "test", "subdir2", "point2"))
+
+        data_utils.combine_point_directories(os.path.join(temp_dir, "test"))
+
+        assert os.path.exists(os.path.join(temp_dir, "test", "combined_folder"))
+        assert os.path.exists(os.path.join(temp_dir, "test", "combined_folder", "subdir1_point1"))
+        assert os.path.exists(os.path.join(temp_dir, "test", "combined_folder", "subdir2_point2"))
+
+
+def test_stitch_images():
+    fovs = ['fov1', 'fov2']
+    chans = ['nuc1', 'nuc2', 'mem1', 'mem2']
+
+    img_data = np.ones((2, 10, 10, 4), dtype="int16")
+    img_data[0, :, :, 1] += 1
+    img_data[0, :, :, 3] += 2
+
+    data_xr = xr.DataArray(img_data, coords=[fovs, range(10), range(10), chans],
+                           dims=["fovs", "rows", "cols", "channels"])
+
+    stitched_xr = data_utils.stitch_images(data_xr, data_xr.shape[2])
```

Comment:
I think having num_col equal to the shape of one of the images makes it harder to figure out what this test is testing, especially since the image data doesn't actually have enough images to make a stitched image of that size. 

I think it would be clearer to read if this was instead image data of [20, 10, 10, 4], and num_cols was 5.
Then you can check that the output shape is [1, 20/5 * 10, 20/4 * 10, 4]